### DISCUSSION TITLE
Pylontech switch off / wake up per Remote control

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#603] [Show and tell]
Somebody asked me whether I had experience remote turning ON/OFF the Pylontech battery. 
I do not (and I can not find the original post.... until spcqike finds it 😉 )

However, I found by coincidence the explanations of how it is done: Using the CONSOLE port of the battery.

![Console Pinout](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/958aacec-d3a8-4eef-abc0-db194987983b)

**connect Pin4 (+5~+12V) with Pin5 (GND) for wake up
Wake up signal must be ≥0.5Sec, current must be between 5~15mA.** 

Here is a good example of somebody using the console port to do the firmware update
[Pylontech US2000C auslesen und Firmware Update via RS232](https://www.youtube.com/watch?v=izZphGWnwn8)

Even nicer, here is somebody explaining (in German) how he remote controls the powering up of the battery:
[Solar – Pylontech Akku (C-Serie) über Konsole aufwecken](https://www.rustimation.eu/index.php/solar-pylontech-akku-ueber-konsole-aufwecken/)
![IMG_8625-225x300](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/3ddeac82-5da4-4075-bfd4-368e199d3432)

Just in case it is of use for somebody.

**Comment by madmartin** [comment]
I mentioned firmware update also on [the Pylontech-battery wiki page](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Pylontech-battery#firmware) (with download source)

**Reply by Manos1966** [reply]
Danke fuer die Info Martin, daraufhin habe ich das Video in den von dir erstellten Bereich platziert.
